install

```diff
@@ -305,19 +305,22 @@ append_line() {
   line="$2"
   file="$3"
   pat="${4:-}"
+  lno=""
 
   echo "Update $file:"
   echo "  - $line"
-  [ -f "$file" ] || touch "$file"
-  if [ $# -lt 4 ]; then
-    lno=$(\grep -nF "$line" "$file" | sed 's/:.*//' | tr '\n' ' ')
-  else
-    lno=$(\grep -nF "$pat" "$file" | sed 's/:.*//' | tr '\n' ' ')
+  if [ -f "$file" ]; then
+    if [ $# -lt 4 ]; then
+      lno=$(\grep -nF "$line" "$file" | sed 's/:.*//' | tr '\n' ' ')
+    else
+      lno=$(\grep -nF "$pat" "$file" | sed 's/:.*//' | tr '\n' ' ')
+    fi
   fi
   if [ -n "$lno" ]; then
     echo "    - Already exists: line #$lno"
   else
     if [ $update -eq 1 ]; then
+      touch "$file"
       echo >> "$file"
```

Comment:
Thanks. I think we should rewrite these two lines to

```sh
[ -f "$file" ] && echo >> "$file"
```

- Because we don't need to add a blank line above the line we're about to add if the file doesn't exist at the moment
- and `touch` is not necessary as `>>` will create the file anyway

Reply:
Thanks for the suggestion, I changed the code to that.